### DISCUSSION TITLE
Warning 20231023 - pacman -Syu

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#1465] [General]
### Edited 20231112: Fixed by 10.0.5

Upgrading `dhcpcd` to 10.0.4 breaks WiFi connection.

To skip: `pacman -Syu --ignore dhcpcd`

**Comment by kan-ibal** [comment]
Should somebody did not read or forgot the warning :)  please use following recovery procedure:
```
# pacman -Rd dhcpcd
# pacman -U /var/cache/pacman/pkg/dhcpcd-10.0.3-1-aarch64.pkg.tar.xz
```

**Reply by R101G** [reply]
Thanks for this, and just to add that this breaks wifi even if you are using a fixed IP!

There is a z missing from the end of the second command line btw.

**Comment by klassikmann** [comment]
/Speculation
In the archlinuxarm forum the problem is mentioned possibly: https://archlinuxarm.org/forum/viewtopic.php?f=15&t=16640
But there is no reply.
Looking in the upstream-git I found an issue which could fit already closed but after the release of 10.0.4:
https://github.com/NetworkConfiguration/dhcpcd/issues/255
If this issue is the cause of breaking the WiFi connection we have to wait until the next relelase. 
/End Speculation

**Reply by klassikmann** [reply]
There is a new issue opened yesterday in the upstream-git:
"dhcpcd 10.0.4 --enable-privsep fails to fork in the background"
[https://github.com/NetworkConfiguration/dhcpcd/issues/260](url)

**Comment by R101G** [comment]
The rAudio wifi seems a bit fragile at the moment, and I can see why some new users have problems. My advice: if it connects first time, leave it alone. Any changes I try to make lead to problems. It was networkd when I last used it. I see we are now back to netctl.

**Comment by klassikmann** [comment]
fyi: dhcpcd 10.0.5 claiming to solve the issue has been released yesterday by the maintainer in github but it's not yet available in the archlinuxarm repositories.

**Reply by rern** [reply]
Confirm it fixes the issue.

By the way, update 20231111 has put the package in ignore list. Once 10.0.5 released on Arch Linux Arm:
- `sed -i -E 's/(IgnorePkg   =).*/#\1/' /etc/pacman.conf`
- Or wait for next update

**Reply by klassikmann** [reply]
10.0.5 is  available in archlinuxarm now.
